.github/actions/publish-nuget/action.yml

```
@@ -92,19 +102,20 @@ runs:
       # * -p:IncludeSymbols=true -p:SymbolPackageFormat=snupkg are needed to generate symbol packages:
       #   https://docs.microsoft.com/en-us/nuget/create-packages/symbol-packages-snupkg.
       run: |
-        $noWarn = @('NU5104') + @'
+        $noWarn = @('NU5104', 'NU5100') + @'
         ${{ inputs.dotnet-pack-ignore-warning }}
         '@.Split() | ? { $_ }
 
-        $arguments = @(
+        New-NugetPackages @(
             "--configuration:Release",
             "--warnaserror",
             "--no-restore",
             "--no-build",
-            "--output:artifacts",
+            "--output:" + (Join-Path $PWD artifacts),
             "--verbosity:${{ inputs.verbosity }}",
             "-p:NuGetBuild=true",
             "-p:Version=${{ steps.setup.outputs.publish-version }}",
+            "-p:NuspecProperties=version=${{ steps.setup.outputs.publish-version }}",
             "-p:GenerateDocumentationFile=True",
             "-p:NoWarn=$($noWarn -join '%3B')",
```

```suggestion
            "-p:NoWarn=$($noWarn -join '%3B')", # Escape the semicolon to prevent the statement from terminating here.
```
Or something along the lines.

---

If you look through the scripts in yml, there are no inline comments anywhere. I'm not trusting that the lines won't get wrapped, se let's not start now. Also I don't think this needs to be explained here specifically when '%3B' is used all over the repo.

---

It wasn't obvious for me. Obviously, once you put on your escape goggles 🥽 you can see semicolons everywhere.

I understand your concern with the inline comment. OK then.